core/src/main/kotlin/org/jetbrains/research/testspark/core/utils/CommandLineRunner.kt

```diff
@@ -32,17 +33,15 @@ class CommandLineRunner {
                     .redirectErrorStream(true)
                     .start()
             }
-
             val reader = BufferedReader(InputStreamReader(process.inputStream))
             var line: String?
 
             while (reader.readLine().also { line = it } != null) {
-                errorMessage += line
+                executionMsg += line
```

Comment:
By the way, this implementation does not respect line breaks. Therefore, if you noticed, we have error stack traces in a single line.

We need:
```kotlin
val sep = System.lineSeparator();
while (..) {
  executionMsg += "$line$sep"
}
```